// @ant-design/react-native/lib/locale-provider/fa_IR.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _fa_IR = require('../date-picker-view/locale/fa_IR');

var _fa_IR2 = _interopRequireDefault(_fa_IR);

var _fa_IR3 = require('../date-picker/locale/fa_IR');

var _fa_IR4 = _interopRequireDefault(_fa_IR3);

var _fa_IR5 = require('../input-item/locale/fa_IR');

var _fa_IR6 = _interopRequireDefault(_fa_IR5);

var _fa_IR7 = require('../list-view/locale/fa_IR');

var _fa_IR8 = _interopRequireDefault(_fa_IR7);

var _fa_IR9 = require('../modal/locale/fa_IR');

var _fa_IR10 = _interopRequireDefault(_fa_IR9);

var _fa_IR11 = require('../pagination/locale/fa_IR');

var _fa_IR12 = _interopRequireDefault(_fa_IR11);

var _fa_IR13 = require('../picker/locale/fa_IR');

var _fa_IR14 = _interopRequireDefault(_fa_IR13);

var _fa_IR15 = require('../search-bar/locale/fa_IR');

var _fa_IR16 = _interopRequireDefault(_fa_IR15);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

exports['default'] = {
    locale: 'fa',
    DatePicker: _fa_IR4['default'],
    DatePickerView: _fa_IR2['default'],
    InputItem: _fa_IR6['default'],
    Modal: _fa_IR10['default'],
    Pagination: _fa_IR12['default'],
    Picker: _fa_IR14['default'],
    SearchBar: _fa_IR16['default'],
    ListView: _fa_IR8['default']
};
module.exports = exports['default'];
